testing/model-mesh/protelion.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+source "$THIS_DIR/config.sh"
+
+if [[ "$SCALE_INSTANCES" -eq 0 ]]
+then
+    ./run_toolbox.py cluster set-scale ${INSTANCE_TYPE} ${INSTANCE_COUNT}
+fi
+
+for step in ${EXEC_LIST}
+do
+    bash $THIS_DIR/${step}
```

Comment:
you need stop here on error, you can do it with this, instead of forking
```
source $THIS_DIR/${step}
```

Reply:
Fixed (squashed)